src/features/events/components/EventParticipantsList.tsx

```diff
@@ -0,0 +1,181 @@
+import { FC } from 'react';
+import { GridColDef } from '@mui/x-data-grid-pro';
+import { Box, Button, Typography } from '@mui/material';
+
+import AddPersonButton from './AddPersonButton';
+import EventDataModel from 'features/events/models/EventDataModel';
+import messageIds from 'features/events/l10n/messageIds';
+import noPropagate from 'utils/noPropagate';
+import ParticipantListSection from 'features/events/components/ParticipantListSection';
+import theme from 'theme';
+import { useMessages } from 'core/i18n';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import ZUIAvatar from 'zui/ZUIAvatar';
+import ZUIRelativeTime from 'zui/ZUIRelativeTime';
+
+interface EventParticipantsListProps {
+  data: ZetkinEvent;
+  model: EventDataModel;
+  orgId: number;
+}
+
+const EventParticipantsList: FC<EventParticipantsListProps> = ({
+  data,
+  model,
+  orgId,
+}) => {
+  const messages = useMessages(messageIds);
+
+  const columns: GridColDef[] = [
```

Comment:
Sorry for not realizing this sooner, but now that I see how everything's put together, it seems weird to me to define this out here. I feel like the columns should be defined inside of the component that renders the grid, i.e. in `ParticipantListSection`. How does that sound to you?